packages/docs/src/stories/templates/header.stories.ts

```diff
@@ -243,12 +279,12 @@ export const SampleB = {
         <div class="flex items-center">
           <sd-navigation-item href="javascript:void(0)">
             <div class="w-4 h-4 flex items-center">
-              <sd-icon title="Unsere weiteren Auftritte" name="system/website" class="text-xl absolute -ml-1"></sd-icon>
+              <sd-icon name="system/website" label="Website" class="text-xl absolute -ml-1"></sd-icon>
             </div>
           </sd-navigation-item>
           <sd-navigation-item onclick="alert('This could open a search bar')">
             <div class="w-4 h-4 flex items-center">
-              <sd-icon title="Suche" name="system/magnifying-glass" class="text-xl absolute -ml-1"></sd-icon>
+              <sd-icon name="system/magnifying-glass" label="Magnifying glass" class="text-xl absolute -ml-1"></sd-icon>
```

Comment:
Same as above. Please change labels here – think what you would have written here as text.

Reply:
Updated